mammamiradio/static/base.css

```
@@ -227,6 +227,29 @@ body[data-stopped="true"] .waveform {
     0 0 28px rgba(37, 99, 235, 0.15);
 }
 
+/* ── Accessibility utilities ── */
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.9.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)